### ISSUE TITLE
Error from Languages tag

### ISSUE THREAD

**Issue by rpgoldman** [amestsantim/pycspro#3]
I have been parsing some files from [the DHS program](https://www.dhsprogram.com/) using pycspro, and I get errors when the `.DCF` files have a "Languages" tag like this:

```

[Languages]

```

This yields this error:

```
Traceback (most recent call last):
  File "/Users/rpg/.virtualenvs/habitus-demographics/lib/python3.9/site-packages/transitions/core.py", line 891, in _get_trigger
    event = self.events[trigger_name]
KeyError: 'Languages'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/rpg/projects/habitus/DHS-data/src/dhs_data/load_data.py", line 146, in <module>
    main(dir)
  File "/Users/rpg/projects/habitus/DHS-data/src/dhs_data/load_data.py", line 133, in main
    dataset = DHSDataset(dirname)
  File "/Users/rpg/projects/habitus/DHS-data/src/dhs_data/load_data.py", line 52, in __init__
    self.load_dir()
  File "/Users/rpg/projects/habitus/DHS-data/src/dhs_data/load_data.py", line 65, in load_dir
    self.parsed_dictionary = self.dictionary_parser.parse()
  File "/Users/rpg/projects/habitus/pycspro/pycspro/DictionaryParser.py", line 236, in parse
    model.trigger(section_name, section_name, items)
  File "/Users/rpg/.virtualenvs/habitus-demographics/lib/python3.9/site-packages/transitions/core.py", line 897, in _get_trigger
    raise AttributeError("Do not know event named '%s'." % trigger_name)
AttributeError: Do not know event named 'Languages'.
```

The simple fix is to remove the `[Languages]` tag -- since it's empty, I can't imagine it's important. But I thought I would mention this as something that might be fixed. I'm afraid I don't know enough about the parsing machinery to do it myself.

**Comment by amestsantim** [comment]
I have updated the parser to now recognize [Languages] section in the dictionary and include it in the parse tree.
Ideally, we would use the languages included there to pick out the correct labels from the valuesets but that is not something I have time to do at the moment. At least it does not disrupt the parsing of dictionaries with multiple languages now.
I have published v1.1.0 to pypi which includes these chagnes

**Comment by rpgoldman** [comment]
Thank you @amestsantim !  Really do appreciate this package so much!

> At least it does not disrupt the parsing of dictionaries with multiple languages now.

In my case, the parsing of dictionaries *without* multiple languages was also disrupted!  The DCF files had a `[Languages]` block that was empty! 😜
